### PR TITLE
feat: support public key records

### PR DESCRIPTION
**Pull request by Nebulis**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "unique-names-generator": "^4.2.0"
   },
   "devDependencies": {
-    "@govtechsg/dnsprove": "^2.0.6",
+    "@govtechsg/dnsprove": "^2.1.0",
```
Comment (Nebulis) [inline package.json:13]: need 2.1.1

File: index.ts [hunk 2]
```diff
@@ -52,24 +54,44 @@ const createRecord = ({
   };
 };
 
+//
+const isEthereumAddressEvent = (event: CreateEvent): event is DocumentStoreEvent =>
+  !!event.address && !!event.networkId && isFinite(event.networkId);
```
Comment (rjchow) [inline index.ts:59]: runtypes would be cool here but optional
it's got pattern matching :D

File: README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+## Test
+
+1. export DOMAIN=sandbox.openattestation.com
+1. export HOSTED_ZONE_ID=<hosted_zone_id>
+1. export AWS_ACCESS_KEY_ID=<access_key>
+1. export AWS_SECRET_ACCESS_KEY=<secret_key>
+1. npx serverless deploy
```
[the remark this answers is not in the document]
Comment (Nebulis) [reply README.md:7]: overwrite, will add a note

File: index.ts (hunk 2 above)
Comment (Nebulis) [reply index.ts:59]: yup but I'm lazy :see_no_evil:

**Pull request merged**